test/functionalTest/cases/0787_cprs_full_functional/update_three_found_on_cps_two_local_four_not_found_in_ngsiv2.test

```diff
@@ -0,0 +1,970 @@
+echo
+
+
+echo "09. Register E1/T1/A3 in CB, provApp: CP3"
+echo "========================================="
+payload='{
+  "contextRegistrations": [
+  {
+    "entities": [
+      {
+         "type": "T1",
+         "isPattern": "false",
+         "id": "E1"
+      }
+    ],
+    "attributes": [
+      {
+        "name": "A3",
+        "type": "string",
+        "isDomain": "false"
+      }
+    ],
+    "providingApplication": "http://localhost:'${CP3_PORT}'/v1"
+    }
+ ],
+ "duration": "P1M"
+}'
+orionCurl --url /v1/registry/registerContext --payload "$payload"
+echo
+echo
+
+
+echo "10. Register E2/T1/A1 in CB, provApp: CP1"
+echo "========================================="
+payload='{
+  "contextRegistrations": [
+  {
+    "entities": [
+      {
+         "type": "T1",
+         "isPattern": "false",
+         "id": "E2"
+      }
+    ],
+    "attributes": [
+      {
+        "name": "A1",
+        "type": "string",
+        "isDomain": "false"
+      }
+    ],
+    "providingApplication": "http://localhost:'${CP1_PORT}'/v1"
+    }
+ ],
+ "duration": "P1M"
+}'
+orionCurl --url /v1/registry/registerContext --payload "$payload"
+echo
+echo
+
+
+echo "11. Register E2/T1/A2 in CB, provApp: CP2"
+echo "========================================="
+payload='{
+  "contextRegistrations": [
+  {
+    "entities": [
+      {
+         "type": "T1",
+         "isPattern": "false",
+         "id": "E2"
+      }
+    ],
+    "attributes": [
+      {
+        "name": "A2",
+        "type": "string",
+        "isDomain": "false"
+      }
+    ],
+    "providingApplication": "http://localhost:'${CP2_PORT}'/v1"
+    }
+ ],
+ "duration": "P1M"
+}'
+orionCurl --url /v1/registry/registerContext --payload "$payload"
+echo
+echo
+
+
+echo "12. Register E2/T1/A3 in CB, provApp: CP3"
+echo "========================================="
+payload='{
+  "contextRegistrations": [
+  {
+    "entities": [
+      {
+         "type": "T1",
+         "isPattern": "false",
+         "id": "E2"
+      }
+    ],
+    "attributes": [
+      {
+        "name": "A3",
+        "type": "string",
+        "isDomain": "false"
+      }
+    ],
+    "providingApplication": "http://localhost:'${CP3_PORT}'/v1"
+    }
+ ],
+ "duration": "P1M"
+}'
+orionCurl --url /v1/registry/registerContext --payload "$payload"
+echo
+echo
+
+
+echo "13. Register E2/T1/A4 in CB, provApp: CP3 (E2-A4 does not exist in CP3)"
+echo "======================================================================="
+payload='{
+  "contextRegistrations": [
+  {
+    "entities": [
+      {
+         "type": "T1",
+         "isPattern": "false",
+         "id": "E2"
+      }
+    ],
+    "attributes": [
+      {
+        "name": "A4",
+        "type": "string",
+        "isDomain": "false"
+      }
+    ],
+    "providingApplication": "http://localhost:'${CP3_PORT}'/v1"
+    }
+ ],
+ "duration": "P1M"
+}'
+orionCurl --url /v1/registry/registerContext --payload "$payload"
+echo
+echo
+
+
+echo "14. Update/APPEND E1/T1/A4 in CB"
```

Comment:
This is an error inherited from the original functest.
If you look at it closely, you sere that step 14 is part of the preparation, just creating E1/T1/A4 in CB.  Step 15 is the forwarding step, that must be in V2, and it is, of course.

Fixing the error in the Preamble, for both files (this one and the original).

Fixed in d7bb634